libs/dh/market-participant/actors/feature-actors/src/lib/dh-actor.ts

```diff
@@ -16,6 +16,10 @@
  */
 import type { ResultOf } from '@graphql-typed-document-node/core';
 
-import { GetActorsDocument } from '@energinet-datahub/dh/shared/domain/graphql';
+import {
+  GetActorByIdDocument,
+  GetActorsDocument,
+} from '@energinet-datahub/dh/shared/domain/graphql';
 
 export type DhActor = ResultOf<typeof GetActorsDocument>['actors'][0];
+export type DhActorExtended = ResultOf<typeof GetActorByIdDocument>['actor'];
```

Comment:
I think the list actor, and detailed actor is the same type.

Reply:
You're right, they are the same type but the GraphQL queries select different Actor properties for their respective views. Så using the `Actor` type for both will mean that a view has access to more `Actor` properties than what might be selected by a query.

That's how I understand the type - query relationship in GraphQL. Is it correct or am I missing something? :)

Reply:
Ah ofc, you are right :-)